src/databricks/labs/blueprint/wheels.py

```diff
@@ -215,22 +215,61 @@ class WheelsV2(AbstractContextManager):
 
     __version: str | None = None
 
-    def __init__(self, installation: Installation, product_info: ProductInfo, *, verbose: bool = False):
+    def __init__(
+        self, installation: Installation, product_info: ProductInfo, *, verbose: bool = False, no_deps: bool = True
```

Comment:
```suggestion
        self, installation: Installation, product_info: ProductInfo, *, verbose: bool = False
```
No deps has to be a flag on private method. Don't change the constructor, it's not applicable in this case.